fluentd-modsecurity/Dockerfile

```diff
@@ -0,0 +1,37 @@
+FROM alpine/git:v2.24.1 as builder
+
+WORKDIR /
+RUN git clone https://github.com/bitsofinfo/fluentd-modsecurity.git
+
+FROM fluent/fluentd:v1.9.3-debian-1.0
+
+# Use root account to use apt
+USER root
+
+COPY --from=builder /fluentd-modsecurity /plugins/fluentd-modsecurity
+
+ENV BUILD_DEP="make gcc g++ libc-dev"
+# below RUN includes plugin as examples elasticsearch is not required
+# you may customize including plugins as you wish
+RUN buildDeps=$BUILD_DEP \
+ && apt-get update \
+ && apt-get install -y --no-install-recommends make=4.2.1-1.2 gcc=4:8.3.0-1 g++=4:8.3.0-1 \
+ && gem install fluent-plugin-elasticsearch:4.0.7 \
+ && gem install fluent-plugin-modsecurity:0.1.4 \
+ && gem sources --clear-all \
+ && SUDO_FORCE_REMOVE=yes \
+    apt-get purge -y --auto-remove \
+                  -o APT::AutoRemove::RecommendsImportant=false \
+                  $buildDeps \
+ && rm -rf /var/lib/apt/lists/* \
+ && rm -rf /tmp/* /var/tmp/* /usr/lib/ruby/gems/*/cache/*.gem
```

Comment:
Prefer using 
- `set -ex`: so we can see the in between steps of the run
- `apk add --no-cache --virtual xxx`: so you can install without cache and `apk del xxx` the bin installed for the build only
https://github.com/SocialGouv/docker/blob/v0.29.0/azure-db/Dockerfile#L10-L29

Reply:
- set -ex -> OK
- apk add --no-cache --virtual xxx -> On utilise une Debian donc pas d' "apk". Les commandes "apt-get update", "apt-get purge" et "rm -rf /var/lib/apt/lists/*" sont équivalentes .

Reply:
Test CI -> OK